src/app/features/user/my-products/components/products-table/products-table.component.html

```diff
@@ -19,8 +19,8 @@
       <td>{{ product.rating }}</td>
       <td class="actions-cell">
         <div>
-          <ile-button [text]="'Edit'" (clicked)="editProductEvent.emit(product.id)"></ile-button>
-          <ile-button [text]="'Delete'" (clicked)="deleteProductEvent.emit(product.id)"></ile-button>
+          <ile-button [text]="'Edit'" (clicked)="$event.stopPropagation(); editProductEvent.emit(product.id)"></ile-button>
```

Comment:
What was the side effect that you blocked using stopPropagation?

Reply:
I made the table row clickable and will navigate to the product page. And when you click on the Edit button it is also navigate although we don't want to navigate.

Reply:
Great!